lib/filewatcher.rb

```diff
@@ -37,7 +36,7 @@ def watch(&on_update)
 
     @on_update = on_update
     @keep_watching = true
-    yield('', '') if @immediate
+    yield({ "" => "" }) if @immediate
```

Comment:
This change is irrelevant. Also, we're using RuboCop and here is its config in the repository.

Reply:
Hm could you clarify?

The specs originally had this: `it { is_expected.to eq [['', '']] }`

So I updated it to `it { is_expected.to eq [{ "" => "" }] }`, I thought that seemed like the most consistent solution

I also briefly considered yielding an empty hash. But I realized that that would cause unnecessary breaking changes, here's what I was thinking:

```ruby
# currently with immediate: true, every: false
fw.watch do |filename, event|
  # filename = "", event = ""
end

# batching changes, yielding { "" => "" }
fw.watch do |changes|
  changes.first(1).each do |filename, event|
    # filename = "", event = ""
  end
end

# batching changes, yielding {}
fw.watch do |changes|
  changes.first(1).each do |filename, event|
    # never runs. user must handle `changes = {}` with a special case
  end
end

# batching changes, yielding "", ""
fw.watch do |changes|
  # changes = ""
end
```

And oops, I will run Rubocop! (Thank you for fixing the gemspec! I had a hard time getting Rubocop to run the other day, but it works now 💪)

Reply:
Sorry, my bad, I didn't notice `,` and `=>`.

I like Symbol keys so `=>` is like trigger for me. :sweat_smile: 

Yes, I meant `yield '' => ''` would be the coolest variant. And are you sure that RuboCop is OK with curly braces here? Usually it ask to omit them when a Hash is a single argument. I'm pretty laze to check it myself, but I will if you confirm.

Reply:
And we should setup RuboCop CI task… oh, it's already here! And it's green! OK, sorry. I'll check RuboCop rules.

Reply:
I found: https://github.com/rubocop-hq/rubocop/issues/7641

I think it's resolved. Sorry.